rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -0,0 +1,160 @@
+import React, {useEffect, useState} from "react";
+import Xarrow from "react-xarrows";
+import '../stylesheets/LandingPageGraph.css'
+
+//Loading Components
```

Comment:
Use formatter in code, configure Code to auto-format. Use Google style if it can deal with react.